joblib/test/test_numpy_pickle.py

```diff
@@ -787,3 +787,23 @@ def test_pathlib():
         nose.tools.assert_equal(numpy_pickle.load(filename), value)
         numpy_pickle.dump(value, filename)
         nose.tools.assert_equal(numpy_pickle.load(Path(filename)), value)
+
+
+@with_numpy
+def test_non_contiguous_array_pickling():
+    arrays = [
+        np.asarray(np.arange(10000000).reshape((100, 500, 200)),
```

Comment:
Do you need big arrays like this? While you are at it, you may as well use np.ones or np.random.randn with a shape argument, it will simplify the line creating the array.